src/hooks/accounts/resolvers/useAccountsMutationResolvers.tsx

```diff
@@ -1,39 +1,39 @@
-import { ApolloCache, ApolloClient, NormalizedCacheObject, Resolver } from '@apollo/client'
-import { find } from 'lodash';
-import { useCallback, useEffect, useRef } from 'react';
-import { SetActiveAccountMutationVariables } from '../mutations/useSetActiveAccountMutation'
+import { Resolver } from '@apollo/client';
+import { useEffect, useRef } from 'react';
 import log from 'loglevel';
-import { useSetActiveAccount } from '../useSetActiveAccount';
 import { useSetActiveAccountMutationResolver } from './useSetActiveAccountMutationResolver';
 
 /**
  * Updates the resolver fn ref each time the given resolver fn changes.
- * 
+ *
  * This is necessary to reflect changes made to functions within hooks
- * within the Apollo resolvers configuration. 
+ * within the Apollo resolvers configuration.
  */
 export const useResolverToRef = (resolver: Resolver, name?: string) => {
-    const resolverRef = useRef(resolver);
-    // when the resolver changes, update the ref
-    useEffect(() => { resolverRef.current = resolver }, [resolver]);
+  const resolverRef = useRef(resolver);
```

Comment:
I _think_ that we might get rid of `useResolverToRef` since i've updated how resolvers are set on the apollo instance after the resolvers update (e.g. dependencies have loaded). But thats a separate topic.

Reply:
I added todo comment to the code to check if it's possible to remove this